config/services.yaml

```diff
@@ -200,3 +200,9 @@ services:
     App\Controller\Api\DatasetPublicationController:
         bind:
             FOS\ElasticaBundle\Persister\ObjectPersister $objectPersister: "@fos_elastica.object_persister.pelagos.dataset"
+
+    App\Util\MailerLoggerUtil:
+        arguments: ["@logger"]
+        tags:
+            - { name: monolog.logger, channel: mailer }
```

Comment:
I think this is the old way of how we used to do it, we can use bind to the respective logger channel